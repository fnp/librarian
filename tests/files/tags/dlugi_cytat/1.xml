<dlugi_cytat><akap> blok-akapitow-cytatu</akap></dlugi_cytat>
