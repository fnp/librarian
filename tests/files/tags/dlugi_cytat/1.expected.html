<div id="book-text"><blockquote>
<a href="#f1" class="wl-num">1</a><p class="wl paragraph" id="f1">blok-akapitow-cytatu</p>
</blockquote></div>
